lib/tasks/access_and_permissions.rake

```diff
@@ -0,0 +1,52 @@
+require "csv"
+
+namespace :permissions do
+  desc "Add an organisation to a document's access permissions list"
+  task :add_organisation_access, %i[document_content_id org_content_id log_file] => :environment do |_, args|
+    document = Artefact.find_by(id: args[:document_content_id])
+
+    if document.nil?
+      message = "Document ID #{args[:document_content_id]} not found, no permissions changed."
+    elsif document.latest_edition.owning_org_content_ids.include?(args[:org_content_id])
+      message = "Organisation already has permission to access the document with ID - #{document.id}"
```

Comment:
Great idea!

Reply:
Since this rake task can be called independently of the bulk one, I think it might still be a good idea to include the org ID in the log messages here.